src/components/UI/index.js

```diff
@@ -0,0 +1,9 @@
+import { TR, TD, TABLE, TBODY, THEAD } from './table';
+
+export {
```

Comment:
Why do we do this instead of directly importing from a table file?

Reply:
I did this so if we ever have any other components, they are all importable from a single file.

Reply:
Personally I think we should come back at some point and have a look at the bigger picture to understand how we can make the entire app more readable and not just single bits and pieces.

Reply:
I had a look at some other libraries and the lodash method seems to be popular where you pull in the thing you need like `import { td, tr } from 'ui/table'`, but yeah we can address this in a follow up, doesn't need to block this PR

Reply:
@mattgrill cool cool, let's do away with index.js then. Not sure what we should have the exports as exactly? My preference would be:`TD`, `TR`, `Table`, `TBody`, and `THead`